.github/ISSUE_TEMPLATE/1-bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Read before posting:
+
+- Questions should be posted to https://community.grafana.com
+- Search/filter through opened and closed issues for existing similar bug reports: https://github.com/grafana/grafana/issues?utf8=%E2%9C%93&q=is%3Aissue
+- Checkout FAQ: https://community.grafana.com/c/howto/faq
+- Checkout How to troubleshoot metric query issues: https://community.grafana.com/t/how-to-troubleshoot-metric-query-issues/50
+- Checkout Using Grafana’s Query Inspector to troubleshoot issues: https://community.grafana.com/t/using-grafanas-query-inspector-to-troubleshoot-issues/2630
```

Comment:
This is at least one item too many, any we can drop because they link to one another?

Reply:
The FAQ one doesn't really contain anything

Reply:
Alternatively, the two last ones (troubleshooting metric quiery issues and how to use the query inspector) could be moved into FAQ and removed from here.

Reply:
`Checkout FAQ` should be `check out the FAQ`. Checkout is a noun. Think you can skip the word Checkout for the How to troubleshoot metric query issues and Using Grafana's query inspector. But agree that with the others that the last two links are very similar.